tests/Unit/DataStructures/Test_SpinWeighted.cpp

```diff
@@ -179,13 +179,15 @@ SPECTRE_TEST_CASE("Unit.DataStructures.SpinWeighted",
     test_spinweights<tmpl::front<type_pair>, tmpl::back<type_pair>>();
   });
 
-  SpinWeighted<ComplexDataVector, 1> size_created_spin_weight_1{6};
-  CHECK(size_created_spin_weight_1.data().size() == 6);
+  SpinWeighted<ComplexDataVector, 1> size_created_spin_weight_1{5};
+  CHECK(size_created_spin_weight_1.data().size() == 5);
```

Comment:
Again, is this just adding a test? If so the commit message should clarify that. Maybe "Add test for SpinWeighted.size()"